CHANGES.rst

```diff
@@ -4,4 +4,19 @@ Changelog
 1.0a1 (unreleased)
 ------------------
 
+- Fixed errors for code analysis test of this package
```

Comment:
@macagua please do not clutter the changelog; add only stuff useful to integrators and developers; IMO this is not useful